### ISSUE TITLE
Show old image near new image in diff view

### ISSUE THREAD

**Issue by justanotheranonymoususer** [gitextensions/gitextensions#8379]
## Feature description

In diff view, looking at source code shows the changed lines.
Looking at images, only the new image is displayed.
GitHub has nice options to switch between the new and old image versions.
At least, add some option to show the old image (with a button, or one near the other).

## Environment

- Git Extensions 3.4.2.9737
- Build 082cfd97d09c47346638b1873f5c8cf6ed139382
- Git 2.24.0.windows.2 (recommended: 2.25.1 or later)
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)

**Comment by gerhardol** [comment]
You can switch old new commit by selecting the commit in reverse order. (Select child before parent.)
The GE image viewer is very limited; no resizing etc so many other changes are required for reviewing images to be really useful.

External difftool will be better at comparing images anyway.
In next version (4.0), it is possible to use a custom tool for a specific compare in the diff tab, see #8194. 
Then you can select a specific tool for a specific compare,. I prefer TortoiseGitMerge for text diff but use p4merge for images and semanticmerge at a refactoring.

**Comment by justanotheranonymoususer** [comment]
> You can switch old new commit by selecting the commit in reverse order. (Select child before parent.)

Yes, but not always. If I e.g. filter by title, I can only see a single version.

> External difftool will be better at comparing images anyway.

That's a slightly different use case. Sometimes I prefer not to leave gitextensions.
I understand adding fancy image viewer features is a lot of work, but I think at least adding a button to toggle between the old/new image will be helpful, and is sounds like a change that is not too difficult.

**Comment by gerhardol** [comment]
Some possible enhancements

## Cycle selection

In revision grid, cycle the selections.
 * If one commit selected, explicitly select the first parent (also if several parents)
 * If two or more, handle selections as a queue and put the last selaected as the first selected

## Improve ImageViewer

Toolbar to switch to the first selected. There is no toolbar now when in image mode. This will require a change to the FileViewer to remember the alternative, it currently just presents "a file" without deeper knowledge.
